### ISSUE TITLE
`RichText` crashes with java.lang.NoSuchMethodError: 'androidx.compose.ui.text.style.TextAlign androidx.compose.ui.text.ParagraphStyle.getTextAlign-buA522U()'

### ISSUE THREAD

**Issue by mr3y-the-programmer** [MohamedRejeb/Compose-Rich-Editor#153]
### Problem:
I get the following exception whenever I try to use `RichText` composable in the latest version 1.0.0-beta05
```
java.lang.NoSuchMethodError: 'androidx.compose.ui.text.style.TextAlign androidx.compose.ui.text.ParagraphStyle.getTextAlign-buA522U()'
	at com.mohamedrejeb.richeditor.parser.html.CssDecoder.decodeParagraphStyleToCssStyleMap$richeditor_compose_release(CssDecoder.kt:88)
	at com.mohamedrejeb.richeditor.parser.html.RichTextStateHtmlParser.decode(RichTextStateHtmlParser.kt:198)
	at com.mohamedrejeb.richeditor.model.RichTextState.toHtml(RichTextState.kt:2187)
	at com.mohamedrejeb.richeditor.model.RichTextState$Companion$Saver$1.invoke(RichTextState.kt:2212)
	at com.mohamedrejeb.richeditor.model.RichTextState$Companion$Saver$1.invoke(RichTextState.kt:2209)
	at androidx.compose.runtime.saveable.ListSaverKt$listSaver$1.invoke(ListSaver.kt:35)
	at androidx.compose.runtime.saveable.ListSaverKt$listSaver$1.invoke(ListSaver.kt:33)
	at androidx.compose.runtime.saveable.SaverKt$Saver$1.save(Saver.kt:66)
	at androidx.compose.runtime.saveable.SaveableHolder$valueProvider$1.invoke(RememberSaveable.kt:148)
	at androidx.compose.runtime.saveable.SaveableHolder.register(RememberSaveable.kt:182)
	at androidx.compose.runtime.saveable.SaveableHolder.onRemembered(RememberSaveable.kt:193)
	at androidx.compose.runtime.CompositionImpl$RememberEventDispatcher.dispatchRememberObservers(Composition.kt:1278)
	at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:975)
	at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:996)
	at androidx.compose.runtime.Recomposer.composeInitial$runtime_release(Recomposer.kt:1087)
	at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:3581)
	at androidx.compose.runtime.ComposerImpl$CompositionContextImpl.composeInitial$runtime_release(Composer.kt:3581)
	at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:629)
	at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:615)
	....
```
### Minimal Reproducer:
```
@Composable
fun Sample(
    modifier: Modifier = Modifier
) {
    Box(
        modifier = modifier,
        contentAlignment = Alignment.Center
    ) {
        val richTextState = rememberRichTextState()
        LaunchedEffect(Unit) {
            richTextState.setHtml(/*any html text or plain text*/)
        }
        RichText(
            state = richTextState
        )
    }
}
```
### Additional Info:
Compose-RichEditor version: "1.0.0-beta05"
Compose BOM version: "2023.10.01" (which maps to compose 1.5.4)
Kotlin version: 1.9.20

**Comment by mr3y-the-programmer** [comment]
Update:
Compose version `1.6.0-alpha08` is brought transitively by another dependency which causes this conflict.

**Comment by MohamedRejeb** [comment]
Hi,
Thanks for reporting this issue. Try to use Kotlin 1.9.21 and compose 1.5.* and let me now if it works.

**Comment by mr3y-the-programmer** [comment]
Yes, it works. it looks like there are some breaking changes in compose 1.6.0 which makes the library incompatible with this version

**Comment by MohamedRejeb** [comment]
Yes, compose multiplatform 1.6.0 still in alpha. When they publish a stable release I will update compose to 1.6.0

**Comment by garretyoder** [comment]
This is actually a compose bug and was fixed upstream here https://issuetracker.google.com/issues/320819734

**Comment by garretyoder** [comment]
The backwards compat was fixed in compose [1.6.0-dev1397](https://github.com/JetBrains/compose-multiplatform/releases/tag/v1.6.0-dev1397) today. @MohamedRejeb, it might be worth throwing a note in the readme for the next couple weeks until 1.6 goes stable.
